exercises-toolbox/8-all/corona/loesung.py

```diff
@@ -13,12 +13,15 @@
 spalten = data.dtype.names
 
 plt.bar(data['Datum'], data['Zuwachs_positiver_Testergebnisse_zum_Vortag'], label='Positive Fälle')
-liste = []
+liste = np.array([])
+dortmund = 6.03609
+rki = 5.88250
 for i in range(data['Datum'].size - 7):
-    liste.append(np.mean(data['Zuwachs_positiver_Testergebnisse_zum_Vortag'][i:i+7].astype(int)))
-plt.plot(data['Datum'][7:], liste, 'r.', markersize=3, label='7-Tage Inzidenz')
+    liste = np.append(liste, np.sum(data['Zuwachs_positiver_Testergebnisse_zum_Vortag'][i:i+7].astype(int)))
```

Comment:
Are you calculating a 7day sum? if yes, please don't do this by hand but e.g. use:

```
np.convolve(data['Zuwachs_positiver_Testergebnisse_zum_Vortag'], np.ones(7))
```

Reply:
If we use convolve he should add the 'valid' tag, otherwise it would also calculate a 1,2,3,..,6-day sum at the start and end of the array.